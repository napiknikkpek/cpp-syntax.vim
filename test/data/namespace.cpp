namespace foo {}
inline namespace foo {}
namespace foo::bar::baz {}
