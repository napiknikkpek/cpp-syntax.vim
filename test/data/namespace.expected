aaaaaaaaa.ttt...
aaaaaa.aaaaaaaaa.ttt...
aaaaaaaaa.ttt..ttt..ttt...
